frontend/app/framework/services/title.service.ts

```diff
@@ -18,15 +19,15 @@ export class TitlesConfig {
     }
 }
 
-export const TitleServiceFactory = (titles: TitlesConfig) => {
-    return new TitleService(titles);
+export const TitleServiceFactory = (titles: TitlesConfig, localizer: LocalizerService) => {
+    return new TitleService(titles, localizer);
 };
 
 @Injectable()
 export class TitleService {
     private readonly stack: any[] = [];
 
-    constructor(private readonly titles: TitlesConfig) {
+    constructor(private readonly titles: TitlesConfig, private readonly localizer: LocalizerService) {
```

Comment:
Please use one line for each field that is directly assigned from the ctor. See other classes here.

Reply:
Done